client/extensions/wp-job-manager/app/controller.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal Dependencies
+ */
+import analytics from 'lib/analytics';
+import titlecase from 'to-title-case';
+import { getSiteFragment, sectionify } from 'lib/route';
+import { renderWithReduxStore } from 'lib/react-helpers';
+import WPJobManager from './main';
+
+export function renderTab( context, component, tab = '' ) {
+	const siteId = getSiteFragment( context.path );
+	const basePath = sectionify( context.path );
+	let baseAnalyticsPath;
+
+	if ( siteId ) {
+		baseAnalyticsPath = `${ basePath }/:site`;
+	} else {
+		baseAnalyticsPath = basePath;
+	}
+
+	let analyticsPageTitle = 'WP Job Manager';
+
+	if ( tab.length ) {
+		analyticsPageTitle += ` > ${ titlecase( tab ) }`;
+	} else {
+		analyticsPageTitle += ' > Job Listings';
+	}
+
+	analytics.pageView.record( baseAnalyticsPath, analyticsPageTitle );
+
+	renderWithReduxStore(
+		<WPJobManager tab={ tab }>
+			{ React.createElement( component ) }
```

Comment:
It doesn't error, but when I'd modified one of the components to actually render something, nothing rendered. When I changed it back, it did. Did you try to render any content?